### PR TITLE
data: PrasunKundu

### PR DESCRIPTION
**Pull request by PrasunKundu**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5760"></a>

### CODE REVIEW

File: data/PrasunKundu.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ "name": Prasun Kundu",
```
Comment (imbeshat) [inline data/PrasunKundu.json:1]:
```suggestion
{ "name":  "Prasun Kundu",
```

File: data/PrasunKundu.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ "displayStatsPublic": true,
```
Comment (imbeshat) [inline data/PrasunKundu.json:2]:
```suggestion
```

Comment (imbeshat) [reply data/PrasunKundu.json:2]: not required.

File: data/PrasunKundu.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ "type": "personal",
+ "bio": "I am a fullstack developer and wanna be a digital nomad",
+"avatar": "https://github.com/PrasunKundu",
```
Comment (imbeshat) [inline data/PrasunKundu.json:5]:
```suggestion
```

Comment (imbeshat) [reply data/PrasunKundu.json:5]: not required

File: data/PrasunKundu.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+{
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/_praasun/",
+      "icon": "FaInstagram"
+    },
+{
+      "name": "Follow me on Github",
+      "url": "https://github.com/PrasunKundu",
+      "icon": "FaGithubAlt"
+    },
```
Comment (imbeshat) [inline data/PrasunKundu.json:21]:
```suggestion
    }
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/PrasunKundu.json (hunk 4 above)
Comment (PrasunKundu) [inline data/PrasunKundu.json:21]: + ]
+}

File: data/PrasunKundu.json (hunk 1 above)
Comment (PrasunKundu) [inline data/PrasunKundu.json:1]: { "name": "Prasun Kundu",

File: data/PrasunKundu.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{ "name": Prasun Kundu",
```
Comment (Pradumnasaraf) [inline data/PrasunKundu.json:1]:
```suggestion
{ "name": "Prasun Kundu",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

There is a missing `"`. Fix that to make the Pull Request ready for merge

File: data/PrasunKundu.json (hunk 5 above)
Comment (PrasunKundu) [reply data/PrasunKundu.json]: Hey , i have made tha changes kindly check

**Review by vedprakashnautiyal — APPROVED** [review]
LGTM 🌱

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

File: data/PrasunKundu.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+{
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/_praasun/",
+      "icon": "FaInstagram"
+    },
+{
+      "name": "Follow me on Github",
+      "url": "https://github.com/PrasunKundu",
+      "icon": "FaGithubAlt"
+    },
```
Comment (Pradumnasaraf) [inline data/PrasunKundu.json:19]:
```suggestion
    }
```

**Review by Pradumnasaraf — DISMISSED** [review]
There is still a chnage need to be done, removing the extra `,` suggested by https://github.com/EddieHubCommunity/LinkFree/pull/5760/files#r1155069528

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by imbeshat — APPROVED** [review]

**Pull request merged**